contrib/single-node.sh

```diff
@@ -18,6 +18,7 @@ sed -i 's#"tcp://127.0.0.1:26657"#"tcp://0.0.0.0:26657"#g' ~/.celestia-app/confi
 sed -i 's/timeout_commit = "5s"/timeout_commit = "1s"/g' ~/.celestia-app/config/config.toml
 sed -i 's/timeout_propose = "3s"/timeout_propose = "1s"/g' ~/.celestia-app/config/config.toml
```

Comment:
Are we intentionally overwriting this for single node networks?

Reply:
yeah, tbh, I don't remember why or when we added this